### ISSUE TITLE
Error during index_metadata()

### ISSUE THREAD

**Issue by KenjiKiuchi** [simonsobs/sotodlib#167]
When I tried to analyze SAT1 data using sotodlib, index_metadata() is terminated due to an error.
  ```
File "~/.venv/lib/python3.6/site-packages/sotodlib/io/load_smurf.py", line 869, in <listcomp>
    ChanAssignments.id.in_([a.id for a in assign_set]))).all()
   AttributeError: 'NoneType' object has no attribute 'id'
```
This happened in the add_new_tuning function in load_smurf.py.
There are none type object in assign_set and this line call None.id, but there is no id in NoneType.
So the function is terminated at "a.id".
Please add exception for null data.

**Comment by kmharrington** [comment]
Yeah, this was supposed to be fixed with this update, I had missed an important continue. https://github.com/simonsobs/sotodlib/commit/84452c065ad0b4a03c247bac07c08b24eae19db0 

Are you still seeing this on the most recent software install? (I do agree that using Jack's database is a good idea long term).
